book/src/css.md

```diff
@@ -0,0 +1,266 @@
+# CSS
+
+When you want to modify the style of your website, you use [CSS](https://de.wikipedia.org/wiki/Cascading_Style_Sheets).
+Since so many people are already familiar with web development, it makes sense that GTK supports its own variant of CSS.
+
+> We will not explain every piece of CSS syntax used in this chapter.
+> If you are new to it or just need a refresher, have a look at the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax).
+
+Let us say we have a button and we want to set its font color to red.
+Every type of widget has a corresponding CSS node.
```

Comment:
A widget can have multiple nodes as well. The combo box arrow from a snippet above is an example, though I don't remember if it's a widget or just a node. But e.g. scrolled window undershoot/overshoot are standalone css nodes.